benches/benches/block_target_gas.rs

```diff
@@ -239,8 +251,554 @@ fn block_target_gas(c: &mut Criterion) {
+    run(
+        "add opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::movi(0x12, 27),
+            op::add(0x10, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "addi opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::addi(0x10, 0x11, 27),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "aloc opcode",
+        group,
+        [op::aloc(0x10), op::jmpb(RegId::ZERO, 0)].to_vec(),
+        vec![],
+    );
+
+    run(
+        "and opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::movi(0x12, 27),
+            op::and(0x10, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "andi opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::andi(0x10, 0x11, 27),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "div opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::movi(0x12, 27),
+            op::div(0x10, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "divi opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::divi(0x10, 0x11, 27),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "eq opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::movi(0x12, 27),
+            op::eq(0x10, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "exp opcode",
+        group,
+        [
+            op::movi(0x11, 23),
+            op::movi(0x12, 11),
+            op::exp(0x10, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "expi opcode",
+        group,
+        [
+            op::movi(0x11, 23),
+            op::expi(0x10, 0x11, 11),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    run(
+        "gt opcode",
+        group,
+        [
+            op::movi(0x11, 100000),
+            op::movi(0x12, 27),
+            op::gt(0x10, 0x11, 0x12),
+            op::jmpb(RegId::ZERO, 0),
+        ]
+        .to_vec(),
+        vec![],
+    );
+
+    {
+        let data = vec![0u8; 32];
+        run(
+            "gtf opcode",
+            group,
+            [
+                op::gtf_args(0x10, RegId::ZERO, GTFArgs::ScriptData),
+                op::jmpb(RegId::ZERO, 0),
+            ]
+            .to_vec(),
+            data,
+        );
+    }
```

Comment:
Moving into a separate section during this PR would be nice because we need to test with any possible `GTFArgs`.

You don't need to test with all `GTFArgs`. But could you create a dedicated follow-up issue only for the `gtf` opcode, please?

We need to find the worst possible getter and use it as the primary price for the GTF opcode.

Reply:
I added it as a sub-requirement of this issue. It looks like a lot of them, so, yeah, I agree it should be done in a separate PR :)

Reply:
@bvrooman We intend to add thorough coverage for `gtf` with another PR.